modules/gapi/src/api/gkernel.cpp

```diff
@@ -20,128 +20,80 @@
 // GKernelPackage public implementation ////////////////////////////////////////
 void cv::gapi::GKernelPackage::remove(const cv::gapi::GBackend& backend)
 {
-    m_backend_kernels.erase(backend);
+    std::vector<std::string> id_deleted_kernels;
+    for (const auto& p : m_id_kernels)
+    {
+        if (p.second.first == backend)
+        {
+            id_deleted_kernels.push_back(p.first);
+        }
+    }
```

Comment:
std::copy_if?

Reply:
most likely transform_if, but we don't have it.

Reply:
Okay